src/lib/templates.js

```diff
@@ -118,19 +122,25 @@ const eventTemplate = ({ event, isEmail, uid }) => {
         class="plugin-calendar-time-date-view"
       >[[moment:time-date-view, local, ${startDate}, ${endDate}, ${allday}]]</a>
     </div>
-    ${event.location.length ? `
+    ${event.external ? `
+    <div class="plugin-calendar-event-location">
+      <i class="fa fa-location-arrow" aria-hidden="true"></i>
+      <span>External: ${event.source}<span>
+    </div>
+    ` : ''}
+    ${event.location ? `
     <div class="plugin-calendar-event-location">
       <i class="fa fa-location-arrow" aria-hidden="true"></i>
       <span>${event.location}<span>
     </div>
     ` : ''}
-    ${event.description.length ? `
+    ${event.description ? `
     <div class="plugin-calendar-event-description">
       <i class="fa fa-info-circle" aria-hidden="true"></i>
       <div>${event.description}</div>
     </div>
     ` : ''}
-    ${event.reminders.length ? `
+    ${!event.external && event.reminders ? `
```

Comment:
Is this a reminder for you to re-add `.length` for these? I'm just a little confused because empty arrays are truthy.